spec.html

```diff
@@ -37,6 +37,151 @@ <h1>Introduction</h1>
   <p>`Promise.any()` accepts an array of promises and returns a promise that is fulfilled by the first given promise to be fulfilled, or rejected with an array of rejection reasons if all of the given promises are rejected.</p>
 </emu-intro>
 
+<emu-clause id="sec-well-known-intrinsic-objects">
+  <h1>Well-Known Intrinsic Objects</h1>
+  <p>Well-known intrinsics are built-in objects that are explicitly referenced by the algorithms of this specification and which usually have realm-specific identities. Unless otherwise specified each intrinsic object actually corresponds to a set of similar objects, one per realm.</p>
+  <p>Within this specification a reference such as %name% means the intrinsic object, associated with the current realm, corresponding to the name. Determination of the current realm and its intrinsics is described in <emu-xref href="#sec-execution-contexts"></emu-xref>. The well-known intrinsics are listed in <emu-xref href="#table-7"></emu-xref>.</p>
+  <emu-table id="table-7" caption="Well-Known Intrinsic Objects">
+    <table>
+      <tbody>
+      <tr>
+        <th>
+          Intrinsic Name
+        </th>
+        <th>
+          Global Name
+        </th>
+        <th>
+          ECMAScript Language Association
+        </th>
+      </tr>
+      <tr>
+        <td>
+          <ins>%AggregateError%</ins>
+        </td>
+        <td>
+          <ins>`AggregateError`</ins>
+        </td>
+        <td>
+          <ins>The `AggregateError` constructor (<emu-xref href="#sec-aggregate-error-constructor"></emu-xref>)</ins>
+        </td>
+      </tr>
+      <tr>
+        <td>
+          <ins>%AggregateError.prototype%</ins>
+        </td>
+        <td>
+          <ins>`AggregateError.prototype`</ins>
+        </td>
+        <td>
+          <ins>The initial value of the `prototype` data property of %AggregateError%</ins>
+        </td>
+      </tr>
+      </tbody>
+    </table>
+  </emu-table>
+</emu-clause>
+
+<emu-clause id="sec-error-objects">
+  <h1>Error Objects</h1>
+  <p>Instances of Error objects are thrown as exceptions when runtime errors occur. The Error objects may also serve as base objects for user-defined exception classes.</p>
+  <p>When an ECMAScript implementation detects a runtime error, it throws a new instance of one of the _NativeError_ objects defined in <emu-xref href="#sec-native-error-types-used-in-this-standard"></emu-xref> or a new instance of %AggregateError%. Each of these objects has the structure described below, differing only in the name used as the constructor name instead of _NativeError_, in the `name` property of the prototype object, in the implementation-defined `message` property of the prototype object, and in the presence of the %AggregateError%-specific `errors` property.</p>
```

Comment:
why is aggregate error not just another one of the native error types?

Reply:
AggregateError has different constructor behavior than NativeErrors as @bakkot mention in #27

Reply:
Hmm, I’m not sure that suggestion is in fact the best solution - it ends up duplicating a lot of spec text. Is there a way we could modify the NativeError approach so it could handle AggregateErrors?

Reply:
I think that it's possible. Will try to figure out